docs/02.API-REFERENCE.md

```diff
@@ -4921,3 +5125,133 @@ jerry_arraybuffer_write (const jerry_value_t value,
 - [jerry_create_arraybuffer](#jerry_create_arraybuffer)
 - [jerry_arraybuffer_write](#jerry_arraybuffer_write)
 - [jerry_get_arraybuffer_byte_length](#jerry_get_arraybuffer_byte_length)
+
+
+## jerry_get_typedarray_type
+
+**Summary**
+
+Get the type of the TypedArray.
+
+The returned type is one of the [jerry_typedarray_class_t](#jerry_typedarray_class_t)
+enum value.
+
+**Prototype**
+
+```c
+jerry_typedarray_class_t
+jerry_get_typedarray_type (jerry_value_t value);
+```
+
+- `value` - TypedArray object to query for type.
+- return
+  - the type of the TypedArray
+  - JERRY_TYPEDARRAY_UNKNOWN if the object was not a TypedArray
+
+**Example**
+
+```c
+{
+  jerry_typedarray_class_t expected_klass = JERRY_TYPEDARRAY_UINT32;
+  jerry_value_t typedarray = jerry_create_typedarray (expected_klass, 25);
+
+  jerry_typedarray_class_t klass = jerry_get_typedarray_type (typedarray);
+
+  // klass is now JERRY_TYPEDARRAY_UINT32
```

Comment:
klass?

Reply:
Yeah, I've used  `klass` as `class` is a keyword in C++. So if someone copies the code into c++ it would still compile without problems.